### ISSUE TITLE
Feature request: add `secret init` command to simplify adding SSM secure parameters

### ISSUE THREAD

**Issue by bvtujo** [aws/copilot-cli#1920]
Our documentation mentions that we plan to add a command to simplify adding secrets.

I imagine that this would work something like the following
```
copilot secret init
Found application myapp registered to the current workspace. 
Which environment would you like this secret to refer to?
> test
   prod
   gamma
   None (don't version by environment)
What would you like to name this secret? db-password
What is the secret's value? **************************
... Creating the secret /copilot/secrets/myapp/test/db-password. 
Would you like to add another value of this secret for a different environment? (Y/n)
...
You can refer to this secret from your services by editing the secrets section of your manifest
to refer to MY_SECRET:/copilot/secrets/myapp/test/db-password
```

We should consider how to prioritize.

**Comment by coatsbj** [comment]
Definitely.  Multiple times now I have forgotten to add the tags when deploying across regions, and the failure is so incredibly obscure it takes forever to realize what you botched!

**Comment by efekarakus** [comment]
Related https://github.com/aws/copilot-cli/issues/2096#issuecomment-817809889

**Comment by efekarakus** [comment]
Hi folks! I'm happy to announce that this command is now available in v1.7.0 🥳 https://github.com/aws/copilot-cli/releases/tag/v1.7.0
